src/qt/test/paymentservertests.cpp

```diff
@@ -43,7 +43,7 @@ static SendCoinsRecipient handleRequest(PaymentServer* server, std::vector<unsig
     // Write data to a temp file:
     QTemporaryFile f;
     f.open();
-    f.write((const char*)&data[0], data.size());
+    f.write((const char*) data.data(), data.size());
```

Comment:
nit: Please remove space after the case here

Reply:
@MeshCollider You can pipe the diff through clang-format to prevent such nits in the future. See dev notes.

Reply:
Fixed, thanks :)